### PR TITLE
Added the official nf-core logo to the README doc.

### PR DESCRIPTION
**Pull request by ignaciot**
Many thanks to contributing to nf-core/nascent!

Please fill in the appropriate checklist below (delete whatever is not relevant). These are the most common things requested on pull requests (PRs).

## PR checklist
 - [X] This comment contains a description of changes (with reason)
 - [ ] If you've fixed a bug or added code that should be tested, add tests!
 - [ ] If necessary, also make a PR on the [nf-core/nascent branch on the nf-core/test-datasets repo]( https://github.com/nf-core/test-datasets/pull/new/nf-core/nascent)
 - [ ] Ensure the test suite passes (`nextflow run . -profile test,docker`).
 - [ ] Make sure your code lints (`nf-core lint .`).
 - [X] Documentation in `docs` is updated
 - [ ] `CHANGELOG.md` is updated
 - [X] `README.md` is updated

**Learn more about contributing:** https://github.com/nf-core/nascent/tree/master/.github/CONTRIBUTING.md

Added the official nf-core logo to the nascent transcription pipeline.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# nf-core/nascent
+# ![nfcore/nascent](docs/images/nascent_logo.png)
 
 **Nascent Transcription Processing Pipeline**.
```
Comment (drpatelh) [inline README.md]:
```suggestion

```

Comment (drpatelh) [reply README.md]: Can just remove the line above. Otherwise all good 👍

**Comment by drpatelh** [comment]
Wait! I don't think we can merge this into `master` because the Travis tests may fail and mess up the passing tests for the release. Is there another way to do this @apeltzer ?

**Comment by apeltzer** [comment]
Hmm 🤔 get it to dev and make a minor release ?

**Comment by drpatelh** [comment]
You and @sven1103 were going to do a funky branch creation method to add in some stuff to `tools` recently right? Maybe merging a branch with a single commit that `.travis.yml` doesnt know about was the trick? 

See: https://github.com/nf-core/nf-co.re/issues/125#issuecomment-492979582

**Comment by ignaciot** [comment]
Ahh, that’s true... let me create a new branch from master, add the logo
and update README, and try to do a PR on that. I’ll kill the current PR.

-i

**Pull request closed without merging**